.github/workflows/acceptance-tests.yml

```diff
@@ -439,9 +439,9 @@ jobs:
           go-version-file: 'go.mod'
       - name: Acceptance Tests
         env:
-          MONGODB_ATLAS_PUBLIC_KEY: ${{ secrets.MONGODB_ATLAS_PUBLIC_KEY_CLOUD_DEV_NETWORK }}
```

Comment:
was using the dev network org a mistake here?

Reply:
we have two organizations in cloud-dev that we use for acceptance tests. The main reason was to decrease the changes of hitting the limit of projects that you can have in an org. In addition, the network org has a project with specific configurations needed for some of the test

Reply:
ok, now that we'll be deleting private network I'll undo this change